foxpuppet/browser/window.py

```diff
@@ -2,43 +2,57 @@
 # License, v. 2.0. If a copy of the MPL was not distributed with this file,
 # You can obtain one at http://mozilla.org/MPL/2.0/.
 
-from foxpuppet.windows import Windows
-
 from selenium.webdriver.common.by import By
 
+from foxpuppet.expected import new_window_is_opened
+from foxpuppet.windows import BaseWindow, WindowManager
+
 
-class BrowserWindow(object):
+class BrowserWindow(BaseWindow):
     _file_menu_button_locator = (By.ID, 'file-menu')
     _file_menu_private_window_locator = (By.ID, 'menu_newPrivateWindow')
     _file_menu_new_window_button_locator = (By.ID, 'menu_newNavigator')
     _nav_bar_locator = (By.ID, 'nav-bar')
     _tab_browser_locator = (By.ID, 'tabbrowser-tabs')
 
     def __init__(self, selenium, *args, **kwargs):
```

Comment:
You shouldn't need to call out the `selenium` argument, it will just be passed through. In fact, if this `__init__` doesn't do anything unique, you can remove it entirely.